src/main/java/com/atlassian/bitbucket/jenkins/internal/model/EmptyBitbucketPage.java

```diff
@@ -0,0 +1,9 @@
+package com.atlassian.bitbucket.jenkins.internal.model;
+
+public final class EmptyBitbucketPage<T> extends BitbucketPage<T> {
```

Comment:
I'm not a fan of this class, it feels like just bloat. Can't we just use a normal `BitbucketPage`? You could add a default constructor that sets the `values` to empty list and `lastPage` to true, and this class can now be removed.

Reply:
I have deleted this